docs/changelog-fragments/216.bugfix.rst

```diff
@@ -0,0 +1 @@
+Added additional error reporting to SFTP write errors -- by :user:`Qalthos`
```

Comment:
Is this one really a bugfix? Do you want to use a fragment number here too, to control ordering in the changelog?